examples/gallery/ui/pages/table_view_page.slint

```diff
@@ -0,0 +1,44 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+import { HorizontalBox, VerticalBox, StandardTableView, GroupBox} from "std-widgets.slint";
+import { GallerySettings } from "../gallery_settings.slint";
+import { Page } from "page.slint";
+
+export global TableViewPageAdapter := {
+    callback sort_ascending(int);
+    callback sort_descending(int);
+    property <[[StandardListViewItem]]> row_data;
```

Comment:
would be nice to have some default stuff there, so it shows in the slint-viewer

A bit unfortunate that there is no soting possible in slint, but that can be improved later.